code_samples/recent_activity/src/Command/DispatchMyFeatureEventCommand.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace App\Command;
+
+use App\Event\MyFeatureEvent;
+use App\MyFeature\MyFeature;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+class DispatchMyFeatureEventCommand extends Command
+{
+    protected static $defaultName = 'app:test:throw-my-feature-event';
+
+    private EventDispatcherInterface $eventDispatcher;
+
+    public function __construct(EventDispatcherInterface $eventDispatcher)
+    {
+        parent::__construct(self::$defaultName);
+        $this->eventDispatcher = $eventDispatcher;
```

Comment:
That's default, and should be omitted.
```suggestion
        $this->eventDispatcher = $eventDispatcher;
        parent::__construct();
```
Additionally, the order of construction for Commands should usually be reversed (than normal). This is due to `$this->configure()` being called internally, which often uses dependencies to describe available options or interact with the user.